### PR TITLE
feat: [maps-routing] new module for maps-routing

### PR DESCRIPTION
**Pull request by suztomo**

### CODE REVIEW

File: versions.txt [hunk 1]
```diff
@@ -586,3 +586,6 @@ grpc-google-cloud-workflows-v1:2.6.0:2.6.0
 proto-google-cloud-workflows-v1beta:0.12.0:0.12.0
 proto-google-cloud-workflows-v1:2.6.0:2.6.0
 google-cloud-dns:2.4.0:2.4.0
+google-maps-routing:0.0.0:0.0.1-SNAPSHOT
```
Comment (meltsufin) [inline versions.txt:589]: Should the versions above also be `-SNAPSHOT`?

Comment (suztomo) [reply versions.txt:589]: Right. I'm merging Release Please's snapshot pull request https://github.com/googleapis/google-cloud-java/pull/8794 first.

Comment (suztomo) [reply versions.txt:589]: Merged. The lines above have "-SNAPSHOT" now.

**Comment by suztomo** [comment]
```
diff --git a/java-maps-routing/google-maps-routing-bom/pom.xml b/java-maps-routing/google-maps-routing-bom/pom.xml
index 198cc242..b[5](https://github.com/googleapis/google-cloud-java/actions/runs/3500494769/jobs/5863222771#step:5:6)01870d 100[6](https://github.com/googleapis/google-cloud-java/actions/runs/3500494769/jobs/5863222771#step:5:7)44
--- a/java-maps-routing/google-maps-routing-bom/pom.xml
+++ b/java-maps-routing/google-maps-routing-bom/pom.xml
@@ -9,[7](https://github.com/googleapis/google-cloud-java/actions/runs/3500494769/jobs/5863222771#step:5:8) +9,7 @@
   <parent>
     <groupId>com.google.cloud</groupId>
     <artifactId>google-cloud-pom-parent</artifactId>
-    <version>0.3.0</version><!-- {x-version-update:google-cloud-pom-parent:current} -->
+    <version>0.4.0-SNAPSHOT</version><!-- {x-version-update:google-cloud-pom-parent:current} -->
     <relativePath>../../google-cloud-pom-parent/pom.xml</relativePath>
   </parent>
 
diff --git a/java-maps-routing/pom.xml b/java-maps-routing/pom.xml
index c375eefa..2ea54[8](https://github.com/googleapis/google-cloud-java/actions/runs/3500494769/jobs/5863222771#step:5:9)e4 100644
--- a/java-maps-routing/pom.xml
+++ b/java-maps-routing/pom.xml
@@ -13,7 +13,7 @@
   <parent>
     <groupId>com.google.cloud</groupId>
     <artifactId>google-cloud-jar-parent</artifactId>
-    <version>0.3.0</version><!-- {x-version-update:google-cloud-jar-parent:current} -->
+    <version>0.4.0-SNAPSHOT</version><!-- {x-version-update:google-cloud-jar-parent:current} -->
     <relativePath>../google-cloud-jar-parent/pom.xml</relativePath>
   </parent>
```

**Comment by suztomo** [comment]
Generation diff succeeded.

Screen Shot 2022-11-21 at 9 33 42 AM

**Comment by suztomo** [comment]
@meltsufin PTAL.

**Review by meltsufin — APPROVED** [review]

**Pull request merged**